### PR TITLE
WRO-4469: added resolveFallback option

### PR DESCRIPTION
**Pull request by hong6316**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Previously, Enact CLI 4.x(includes webpack4) avoided fs-related build errors via using `nodeBuiltins` option.
https://v4.webpack.js.org/configuration/node/#other-node-core-libraries
But after CLI 5.x(includes webpack5), that option didn't work anymore.
https://webpack.js.org/configuration/node/#node

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
As per upgrading webpack v4->v5, node.fs: 'empty' option has been changed the other way.
https://webpack.js.org/migrate/5/#clean-up-configuration
For this reason, we added an additional option to avoid fs-related errors again. - `resolveFallback`

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)

### Comments
Enact-DCO-1.0-Signed-off-by: Taeyoung Hong ([taeyoung.hong@lge.com](mailto:taeyoung.hong@lge.com))

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -68,6 +68,7 @@ The @enact/cli tool will check the project's **package.json** looking for an opt
 * `ri` _[object]_ - Resolution independence options to be forwarded to the [LESS plugin](https://github.com/enactjs/less-plugin-resolution-independence). By default, will use any preset for a specified theme or fallback to sandstone.
 * `screenTypes` _[array|string]_ - Array of 1 or more screentype definitions to be used with prerender HTML initialization. Can alternatively reference a json filepath to read for screentype definitions.  By default, will use any preset for a specified theme or fallback to sandstone.
 * `nodeBuiltins` _[object]_ - Configuration settings for polyfilling NodeJS built-ins. See `node` [webpack option](https://webpack.js.org/configuration/node/).
+* `resolveFallbackBuiltins` _[object]_ - Configuration settings for redirecting module requests when normal resolving fails. See `resolve.fallback` [webpack option](https://webpack.js.org/configuration/resolve/#resolvefallback).
```
Comment (MikyungKim) [inline README.md:71]:
```suggestion
* `resolveFallback` _[object]_ - Configuration settings for redirecting module requests when normal resolving fails. See `resolve.fallback` [webpack option](https://webpack.js.org/configuration/resolve/#resolvefallback).
```

Comment (MikyungKim) [reply README.md:71]: Please rename the option.

File: docs/starting-a-new-app.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ The @enact/cli tool will check the project's **package.json** looking for an opt
 * `ri` _[object]_ - Resolution independence options to be forwarded to the [LESS plugin](https://github.com/enactjs/less-plugin-resolution-independence). By default, will use any preset for a specified theme or fallback to sandstone.
 * `screenTypes` _[array|string]_ - Array of 1 or more screentype definitions to be used with prerender HTML initialization. Can alternatively reference a json filepath to read for screentype definitions.  By default, will use any preset for a specified theme or fallback to sandstone.
 * `nodeBuiltins` _[object]_ - Configuration settings for polyfilling NodeJS built-ins. See `node` [webpack option](https://webpack.js.org/configuration/node/).
+* `resolveFallbackBuiltins` _[object]_ - Configuration settings for redirecting module requests when normal resolving fails. See `resolve.fallback` [webpack option](https://webpack.js.org/configuration/resolve/#resolvefallback).
```
Comment (MikyungKim) [inline docs/starting-a-new-app.md:32]: I think we have the same option list in templates. Please update there too.

Comment (hong6316) [reply docs/starting-a-new-app.md]: OK
https://github.com/enactjs/templates/pull/47

File: README.md (hunk 1 above)
Comment (hong6316) [reply README.md]: OK

File: package.json [hunk 3]
```diff
@@ -49,7 +49,7 @@
     "@babel/preset-react": "^7.16.7",
     "@babel/preset-typescript": "^7.16.7",
     "@babel/runtime": "^7.16.7",
-    "@enact/dev-utils": "^5.0.0-alpha.2",
+    "@enact/dev-utils": "github:enactjs/dev-utils#feature/WRO-4469",
```
Comment (MikyungKim) [inline package.json:52]: Please revert the change.

File: npm-shrinkwrap.json [hunk 4]
```diff
@@ -10362,13 +10362,13 @@
       "integrity": "sha512-iO9ZQHkZxHn4mSakYV0vFHAVDyEOIJQrV2uZ06HxEPcx+mt8swXoZHIbaaJ2crJYFfErySgktuTZ3BeLz+XmFA=="
     },
     "@typescript-eslint/eslint-plugin": {
-      "version": "5.25.0",
-      "resolved": "https://registry.npmjs.org/@typescript-eslint/eslint-plugin/-/eslint-plugin-5.25.0.tgz",
-      "integrity": "sha512-icYrFnUzvm+LhW0QeJNKkezBu6tJs9p/53dpPLFH8zoM9w1tfaKzVurkPotEpAqQ8Vf8uaFyL5jHd0Vs6Z0ZQg==",
+      "version": "5.24.0",
```
Comment (MikyungKim) [inline npm-shrinkwrap.json:10365]: Please revert this change too.
